### PR TITLE
Add sponsorship page

### PR DESCRIPTION
**Pull request by CJEnright**
Screenshot_2019-10-11 BoilerMake

### CODE REVIEW

File: templates/web/pages/sponsors.tmpl [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+			</div>
+		</div>
+		<div class="columns is-mobile is-centered content">
+			<div class="column is-3">
+				<img src={{ static_path "sponsors_twilio.png" }} alt="Twilio">
+			</div>
+			<div class="column is-3">
+				<img src={{ static_path "sponsors_walmart.png" }} alt="Walmart">
+			</div>
+		</div>
```
Comment (colejoh) [inline templates/web/pages/sponsors.tmpl]: You should throw some `alight-items: center` on these.

Comment (CJEnright) [reply templates/web/pages/sponsors.tmpl]: Good call, updated:
Screenshot_2019-10-11 BoilerMake(1)

**Comment by romulus3799** [comment]
Is this up to date with our current visual theme or is that to be implemented?

**Comment by CJEnright** [comment]
We're just gonna keep this as is, it won't be very trafficked during the on season but the info is still worth showing.

**Pull request merged**